README.md

```diff
@@ -94,9 +94,8 @@ The installation was tested on Windows Subsystem for Linux **V1** ([WSL](https:/
 
 2. Install **Node.js** and **npm** from the same terminal. Currently, the latest **Node.js** LTS version compatible with **YarpJS** `master` commit [be28630](https://github.com/robotology/yarp.js/commit/be2863022713ded2fa48909404b43e98b09eeda2) is the LTS: Argon release **Node.js v4.2.2** (refer to https://github.com/robotology/yarp.js/issues/19). Meanwhile, the latest **Node.js** LTS version compatible with **Open MCT** is latest LTS:Fermium version **Node.js v14.17.0**. For this reason we install both releases.
     ```
-    nvm install 4.2.2 --latest-npm          // installs the LTS:Argon version v4.2.2 and latest respective supported npm (v2.14.7)
     nvm install 14.17.0 --latest-npm        // installs latest LTS:Fermium version v14.17.0 and latest respective supported npm (v6.14.13)
-    nvm alias default v4.2.2                // updates the default alias
+    nvm alias default v14.17.0              // updates the default alias
     nvm ls                                  // lists available and active nodejs and npm versions
     ```
```

Comment:
I think that the comment concerning the YarpJS Node.js version restriction can be removed (as far as I understand also from the script), am I right?

Reply:
As discussed, the issue is in the comment above this. Will fix.

Reply:
Fixed README and `npm start`.